modules/benchmark/R/align_data.R

```diff
@@ -43,8 +43,8 @@ align_data <- function(model.calc, obvs.calc, var, align_method = "match_timeste
   coarse.unit <- compare$diff_units[coarse]
   
   # Round to the larger time step (experimental)
-  obvs.calc$round.posix  <- as.POSIXct(round(obvs.calc$posix,  units = coarse.unit))
-  model.calc$round.posix <- as.POSIXct(round(model.calc$posix, units = coarse.unit))
+  obvs.calc$round.posix  <- as.POSIXct(round(obvs.calc$posix),  units = coarse.unit)
+  model.calc$round.posix <- as.POSIXct(round(model.calc$posix), units = coarse.unit)
```

Comment:
This is a tricky one, but I think `units` was correctly inside the `round` call here -- the calculation will dispatch to `round.POSIXt`, which accepts a `units` argument even though none is documented for the `round` generic. 
```suggestion
  obvs.calc$round.posix  <- as.POSIXct(round(obvs.calc$posix,  units = coarse.unit))
  model.calc$round.posix <- as.POSIXct(round(model.calc$posix, units = coarse.unit))
```